package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@clerk/backend": "0.36.0",
```

Comment:
@domitriusclark We should update `backend` as well

Reply:
Thanks for that callout 🤝